OpenEdXMobile/src/main/java/org/edx/mobile/view/AccountFragment.kt

```diff
@@ -0,0 +1,86 @@
+package org.edx.mobile.view
+
+import android.content.Intent
+import android.os.Bundle
+import android.text.TextUtils
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import com.google.inject.Inject
+import org.edx.mobile.BuildConfig
+import org.edx.mobile.R
+import org.edx.mobile.annotation.Nullable
+import org.edx.mobile.base.BaseFragment
+import org.edx.mobile.core.IEdxEnvironment
+import org.edx.mobile.databinding.FragmentAccountBinding
+import org.edx.mobile.deeplink.Screen
+import org.edx.mobile.deeplink.ScreenDef
+import org.edx.mobile.module.prefs.LoginPrefs
+import org.edx.mobile.util.Config
+
+class AccountFragment : BaseFragment() {
+
+    companion object {
```

Comment:
it should be defined just after the data members. in this case, just before the `onCreateView` method.